versions.tf

```diff
@@ -1,7 +1,7 @@
 terraform {
-  required_version = ">= 0.13.0"
+  required_version = ">= 1.4.0, < 2.0.0"
```

Comment:
I'm going to regret this :D

Reply:
Why?

Reply:
It's more or less the same reasons I don't like maximum versions in the provider. We don't actually know that this will be incompatible with Terrafom 2.0. Max versions seem to cause more problems than they prevent. But 2.0 is a ways away, and there's a ticket for an ADR in the eng reserve backlog -- once that's being drafted, we can fight it out :)

Reply:
Is there something in this module that requires Terraform 1.4.0 or higher? If not, I think we can leave the Terraform version constraint as `required_version = ">= 1.0"` (the module's Terraform version constraint was updated recently).